test/e2e_node/e2e_service.go

```diff
@@ -234,10 +239,15 @@ func (es *e2eService) startKubeletServer() (*killCmd, error) {
 		"--config", es.kubeletStaticPodDir,
 		"--file-check-frequency", "10s", // Check file frequently so tests won't wait too long
 		"--v", LOG_VERBOSITY_LEVEL, "--logtostderr",
+		"--network-plugin=kubenet",
```

Comment:
This was added on by some other PR aswell. I am not sure why its showing up as green.